account_bank_statement_import_online_ponto/tests/test_account_bank_statement_import_online_ponto.py

```diff
@@ -142,7 +142,7 @@ def setUp(self):
         )
 
     def test_ponto(self):
-        with self.mock_transaction(), self.mock_header(), self.mock_synchronisation(), self.mock_account_ids():
+        with self.mock_transaction(), self.mock_header(), self.mock_synchronisation(), self.mock_account_ids():  # noqa: B950
```

Comment:
I had to add this here because I couldn't split the line to make it shorter. 

If I wrap it around (), I get an error when running the test because I think that means a different interpretation for python. 
If I split them with \, black reformats it back to this way.

@pedrobaeza do you have any suggestion?

Reply:
I tried that, but black reformats it back to the current version :/